orcid-web/src/main/resources/freemarker/advanced_search.ftl

```diff
@@ -39,52 +39,34 @@
-                                      </div>
-                                    </div>
-                                 </div>
                                 <div class="controls">
-                                    <input type="text" class="input-xlarge" name="affiliationOrg" id="affiliationOrg" [(ngModel)]="input.affiliationOrg">
+                                    <input placeholder="${springMacroRequestContext.getMessage('orcid_bio_search.placeholder.affiliation')}" type="text" class="input-xlarge" name="affiliationOrg" id="affiliationOrg" [(ngModel)]="input.affiliationOrg">
                                 </div>
                             </div>
                             <div class="control-group col-md-6">
                                 <!-- Keyword -->
                                 <label for="familyName" class="control-label">${springMacroRequestContext.getMessage("orcid_bio_search.labelkeywords")}</label>
-                                <div class="popover-help-container">
-                                    <i class="glyphicon glyphicon-question-sign"></i>
-                                    <div id="search-help-keyword" class="popover bottom">
-                                      <div class="arrow"></div>
-                                      <div class="popover-content">
-                                        <p>${springMacroRequestContext.getMessage("orcid_bio_search.popover_help.keyword")}</p>
-                                      </div>
-                                    </div>
-                                 </div>
                                 <div class="controls">
                                         <input type="text" class="input-xlarge" name="keyword" id="keyword" [(ngModel)]="input.keyword">
                                 </div>
                             </div>
                         </div>
                     </fieldset>
-                    <hr>
+                    <hr class="row">
                     <fieldset>
                         <div class="row">
                             <p>${springMacroRequestContext.getMessage("orcid_bio_search.searchorcid")}</p>
                             <div class="control-group">
                                 <!-- Search by ORCID iD -->
                                 <label for="orcid" class="control-label">${springMacroRequestContext.getMessage("orcid_bio_search.labelsearchbyorcid")}</label>
                                 <div class="controls">
-                                    <input type="text" class="input-xlarge" name="orcid" id="orcid" [(ngModel)]="input.text">
+                                    <input  placeholder="XXXX-XXXX-XXXX-XXXX" type="text" class="input-xlarge" name="orcid" id="orcid" [(ngModel)]="input.text">
```

Comment:
@lizkrznarich please let me know if you prefer to have the placeholder `XXXX-XXXX-XXXX-XXXX` in a translation property